plugins/newsdownloader.koplugin/main.lua

```diff
@@ -256,54 +439,130 @@ function NewsDownloader:deserializeXMLString(xml_str)
     -- see: koreader/plugins/newsdownloader.koplugin/lib/LICENSE_LuaXML
     local treehdl = require("lib/handler")
     local libxml = require("lib/xml")
-
-    --Instantiate the object the states the XML file as a Lua table
+    --Instantiate the object that states the XML file as a Lua table
```

Comment:
Comments like these as proper sentences please. You'll see a lot of these kinds of comments from me if you're going to be nitpicking minor stuff. ;-)

Btw, _state_ doesn't feel like the right verb.
```suggestion
    -- Instantiate the object that states the XML file as a Lua table.
```

Reply:
I appreciate the nitpicks!

Note to self: comments as full sentences.